### DISCUSSION TITLE
Submitted URL marked ‘noindex’

### DISCUSSION THREAD

**Discussion by DarkChromaMC** [NamelessMC/Nameless#1946] [Q&A] [answered]
I'm trying to index my website through google console since my namelessmc powered website is not appearing in google searches. I sent google the sitemap generated through the namelessmc sitemap creator, and google search console gave me the following errors:

Submitted URL marked ‘noindex’

The error is occuring on all 34 pages of my website, including the https://example.net/index.php?route=/

**To Reproduce**
Steps to reproduce the behaviour:
1. Go to https://datayze.com/index-noindex
2. Put in https://catcraft.net
3. See verdict results are 'Ambigious'.
3. Now try https://wiki.catcraft.net
4. See verdict results no issues whatsoever.
5. This noindex error is only occuring on the main site using the namelessmc software.

**Expected behaviour**
Google is able to successfully index my website without any problems.

**Screenshots**
![image](https://user-images.githubusercontent.com/68839442/101865320-d3f6ac80-3bda-11eb-9083-dad282ed151b.png)

Here is the sitemap generated for my website:
https://www.catcraft.net/cache/sitemaps/sitemap-index.xml

**Comment by samerton** [comment]
Hi,

This is related to your server configuration, Nameless does not set robot tags.

Your web server is setting a header `X-Robots-Tag: none` which tells search engines not to index your website. Removing this header in your web server configuration will solve the issue

**Comment by DarkChromaMC** [comment]
If you don't mind asking, how do I remove this `X-Robots-Tag: none` header?

**Comment by samerton** [comment, marked as answer]
Issue resolved via Discord

For anyone else that might have the same issue, in this case the nginx configuration file contained a line similar to the following:
`add_header X-Robots-Tag "none"` which was removed
